### DISCUSSION TITLE
【2023.3.1】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#2072] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，各 SIG 负责人有什么议题可以提前准备哦。

**Comment by jsparter** [comment]
#2071 oci registry

**Comment by starnop** [comment]
0.9.2 release progress

**Comment by kakaZhou719** [comment]
3月1日 【会议纪要】


1. zot image 存储位置/kyverno image存储位置。可以考虑GitHub action 制作镜像. 
2. 需要创建sealer app repo，基础镜像的kubefile需要分开,包括，zot registry，docker registry,kyverno image, k0s sealer image 以及sealer app 等。
3. 同步 发版本进展信息，0.9.2。预计3月3号号。
